.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+# Description
```

Comment:
Maybe you can add an additional line which references back to an issue; something like this:

```
This PR fixes #...
```

Apart from that, it looks great! :+1:

Reply:
* Mmh, shouldn't the ID of the issue fixed rather be in the commit message?

* Minor note, could you reduce the size of these headlines? I.e. `###` instead of`#`, the current size may be semantically correct but will just look jarring.

Reply:
The ID should be in the commit message, yes. But why not additionally mention it here? Makes it easier for reviewers.